inference_client/client.py

```diff
@@ -21,7 +21,12 @@ def __init__(
         :param token: An optional user token for authentication.
         """
 
-        self._auth_token = login(token)
+        try:
+            self._auth_token = login(token)
+        except Exception:
+            raise ValueError(
+                f'Invalid or expired auth token. Please try again.'
```

Comment:
Try again for invalid token? We should guild the user to double check the token he is providing